test/e2e/network_test.go

```diff
@@ -27,8 +27,10 @@ func testNetwork(t *testing.T) {
 	// Populate the global test context
 	testCtx, err := NewTestContext()
 	require.NoError(t, err)
-	err = testCtx.waitForWindowsNodes(gc.numberOfNodes, true, false, false)
-	assert.NoError(t, err, "timed out waiting for Windows node")
+	err = testCtx.waitForWindowsNodes(gc.numberOfMachineNodes, false, false, false)
+	assert.NoError(t, err, "timed out waiting for Windows nodes configured from Machines")
+	err = testCtx.waitForWindowsNodes(gc.numberOfBYOHNodes, false, false, true)
```

Comment:
We should introduce wrappers for `waitForWindowsNodes` to clear demarcate whether we are waiting for Machines o BYOH nodes. The code is really hard to read now without committing the function signature to memory. But I won't insist.

Reply:
I'd like to do this when doing the larger refactor around waiting for nodes and Machines https://issues.redhat.com/browse/WINC-620

Reply:
I am fine with that. I updated the story,